libs/entitlement/qa/entitlement-allowed/src/main/plugin-metadata/entitlement-policy.yaml

```diff
@@ -1,2 +1,3 @@
 org.elasticsearch.entitlement.qa.common:
   - create_class_loader
+  - set_https_connection_properties
```

Comment:
Remind me... why do we grant these in here? I thought the QA tests were running through _disallowed_ operations and making sure they all get denied?